group/secretsharing/ss.go

```diff
@@ -0,0 +1,114 @@
+package secretsharing
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/cloudflare/circl/group"
+)
+
+type SecretShare struct {
+	ID    uint
+	Share group.Scalar
+}
+
+type ShamirSS struct {
```

Comment:
```suggestion
type ShamirSecretSplitter struct {
```

I would prefer if exported names were not abbreviated. I don't know if `Splitter` is the best name for this thing, but something like that?

Reply:
I opted by an unexported struct called `secretSharing`.